src/core/server/http/cookie_session_storage.test.ts

```diff
@@ -69,7 +70,14 @@ configService.atPath.mockImplementation((path) => {
     } as any);
   }
   if (path === 'csp') {
-    return new BehaviorSubject({} as any);
+    return new BehaviorSubject({
+      script_src: [],
+      worker_src: [],
+      style_src: [],
+      strict: false,
+      disableEmbedding: false,
```

Comment:
Not specifying `script_src` and others was causing a NPE  in `csp_directives.ts#L47`. As these properties are not optional in the config, I thought it made more sense to adapt the test configs instead of adding test-specific logic in the code.